app/javascript/styles/mastodon/components.scss

```diff
@@ -821,6 +822,7 @@
   a {
     color: $secondary-text-color;
     text-decoration: none;
+    unicode-bidi: isolate;
```

Comment:
I think this shouldn't be applied for all links but links such as mentions and hashtags.

Reply:
But all links are ascii (`https://...`)

Reply:
Not necessarily. Mastodon allows links with arbitrary text from other software.

Reply:
I would wager that the *majority* of links would be ascii URLs and we'll have consistently better results by not making them affect the directionality of the whole post.

Reply:
Is there any way to bring a live instance with these changes so I test multiple possible scenarios?